docs/release-notes.rst

```diff
@@ -1,3 +1,216 @@
+
+------
+
+**Floating IP auto sync and Default Gateway auto sync will not work when hosts are added in an HA cluster using hostnames**
+
+**Workaround:** Manually sync the Floating IPs and Default Gateway to Application DB by clicking on **Sync To DB**
+
+------
+
+**Export db does not work on APIC version 5.0.1**
+
+The user menu on the top-right corner of the F5 ACI ServiceCenter has an option **Export/Import DB > Export DB**, for exporting the F5 ACI ServiceCenter database. This option does not work on APIC versions 5.0.1. This is working as expected on prior versions of the APIC and also on versions 5.0(2e) and higher. 
+
+**Workaround:** Backup the F5 ACI ServiceCenter database via the following steps:
+1. Login to the APIC server via SSH
+
+2. cd /data2/gluster/gv0/F5Networks_F5ACIServiceCenter
+
+3. Use scp, winscp or any other preferred tool to copy out the f5.db file from this location. 
+
+------
+
+**Non-default docker0 configuration on APIC may affect connection to the management IP of Virtual BIG-IP**
+
+The default docker0 bridge IP has the address **172.17.0.1**. If the docker0 bridge IP is updated to a non-default IP followed by APIC cluster reboot, the Virtual BIG-IP login (connection to management IP) will fail from the F5 ACI ServiceCenter.
+
+**Workaround:** After APIC reboots, once again change the docker0 bridge IP to a new value.
+
+------
+
+**Zoom In/Out for Device Discovery Topology diagrams may get stuck on Internet Explorer**
+
+Zoom In and Zoom out of Topology diagrams which open on Topology icon click in every tab (Visibility, Network Management and L4-L7 App Services) may get stuck in Internet Explorer browser. The position of the topology diagram can also not be changed once this issue is observed. 
+
+**Workaround:** Use a different browser than IE.
+
+
+Visibility
+----------
+
+**Visibility report downloads do not work in the Chrome browser version 83 and higher (Fixed in APIC 4.2(4p)+ and 5.0(2e)+)**
+
+F5 ACI ServiceCenter's Visibility tab has a **Download Report** icon for downloading VLAN, VIP and Node table data in CSV format. This download functionality does not work in the Chrome browser version 83 and higher, due to a new security check added by Chrome: https://developers.google.com/web/updates/2020/04/chrome-83-deps-rems#disallow_downloads_in_sandboxed_iframes 
+
+**Fix:** This has been fixed in APIC versions 4.2(4p)+ and 5.0(2e)+.
+
+**Workaround:** To download visibility reports, use a different browser, like Mozilla Firefox, or an earlier version of Google Chrome.
+
+------
+
+**Visibility table fast scrolling on Mozilla firefox for scale configs may result in a blank screen**
+
+This is a known ag-grid issue on Mozilla Firefox browser: https://github.com/ag-grid/ag-grid/issues/2841
+
+**Workaround:** Scroll slowly to prevent this issue. But if this issue is observed, the possible workarounds are:
+
+1. Switch tab from Visibility tab to one of the other tabs and swith back. 
```

Comment:
1. Switch tabs from the Visibility tab to one of the other tabs and then switch back.